README.md

```
@@ -37,6 +38,24 @@ With our component library, you can build a variety of chat use cases, including
+## Build with AI Agents
+
+If you build with an AI coding agent, our [agent skills](https://getstream.io/agent-skills/docs/installation/) teach it how to use this SDK correctly. Install them once:
+
+```bash
+curl -fsSL https://getstream.io/cli.sh | bash
+getstream init
+```
+
+Then reach for the [`/stream-react`](https://getstream.io/agent-skills/docs/skills/stream-react/) skill:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)